FHEM/00_SIGNALduino.pm

```diff
@@ -931,7 +931,8 @@ sub SIGNALduino_Get_FhemWebList {
 			( 
 				(IsDummy($hash->{NAME}) && $_ =~ m/^(?:availableFirmware|raw)/) || 
 				( InternalVal($hash->{NAME},"cc1101_available",0) || (!InternalVal($hash->{NAME},"cc1101_available",0) && $_ !~ /^cc/)) &&  
-				( !IsDummy($hash->{NAME}) && (defined(DevIo_IsOpen($hash))  ||  $_ =~ m/^(?:availableFirmware|raw)/  )) 
+				( !IsDummy($hash->{NAME}) && (defined(DevIo_IsOpen($hash))  ||  $_ =~ m/^(?:availableFirmware|raw)/  )) &&
+				( $_ !~ m/^set_bWidth/ )
 			) 
```

Comment:
Mit der Dropdown-Liste der get-Befehle scheint es aber ein generelles Problem zu geben. Ich habe mittlerweile bemerkt, das es zu der Liste auch noch einen Eintrag "sendraw" hinzufügt, nachdem ich einmal "get ccconf" ausgeführt habe.

Die beiden zusätzlichen Einträge werden vermutlich hierdurch erzeugt:

```
$hash->{ucCmd}->{cmd} = 'sendraw';
$hash->{ucCmd}->{cmd} = "set_bWidth";
```

Reply:
@sidey79
kannst du dir erklären wieso die Auswahllisten die zusätzlichen Einträge nach dem Aufruf besitzen?

Reply:
@sidey79 & @elektron-bbs ,
ich habe nun mal geprüft und rekonstruieren können.
Der Eintrag 
`$hash->{ucCmd}->{cmd} = "set_bWidth";` setzt den Wert welcher nach dem ausführen des Befehls zusätzlich in der Liste der Getbefehle auftaucht. Diesen kann man umbenennen und dann erscheint der umbenannte in der Liste.

WO das ganze aber in den Getliste übernommen wird konnte ich noch nicht finden.

Reply:
Ich habe den Fehler ausfindig gemacht.

Reply:
Perfekt :-) Meine Liste fühle sich schon so dreckig *grins*